### ISSUE TITLE
Redirect to profile after deleting a phone

### ISSUE THREAD

**Issue by mac-chaffee** [Bouke/django-two-factor-auth#159]
https://github.com/Bouke/django-two-factor-auth/blob/master/two_factor/views/core.py#L476

When you delete a phone now, it will take you to `settings.LOGIN_REDIRECT_URL`, which for most people is the homepage.

Would it make more sense to redirect to `reverse('two_factor:profile')`? Like if you want to delete a phone and add a new one? Or delete multiple phones?

**Comment by MarkusH** [comment]
That sounds like a good proposal to me. Tho you can just set `success_url` in the URL config in the `as_view()` call. See https://docs.djangoproject.com/en/dev/topics/class-based-views/#simple-usage-in-your-urlconf for an example with `template_name` instead.

**Comment by mac-chaffee** [comment]
Thanks for making those changes. Looks like the current `PhoneDeleteView` overrides `get_success_url`, so setting `success_url `in my urls.py doesn't work =(

On the off chance that anyone else is looking for a temporary fix:

Extend `PhoneDeleteView`, set `success_url`, override `get_success_url() `to return `success_url`, and add this url above where you included two_factor urls:

```
url(regex=r'^account/two_factor/backup/phone/unregister/(?P<pk>\d+)/$',
        view=NewPhoneDeleteView.as_view(),
        name='phone_delete',),
```

**Comment by Bouke** [comment]
I've resolved this in 3d0d92f; you should be able to override ``success_url`` from ``urls.py`` now.
